### PR TITLE
Revive the MSI installer

### PR DESCRIPTION
**Pull request by zanderz**
Might want some matching server change to not strand people who haven't updated since March 9, when the updater became capable of running .msi

### CODE REVIEW

File: packaging/windows/WIXInstallers/KeybaseApps/KeybaseApps.wxs [hunk 1]
```diff
@@ -262,7 +263,7 @@
     NOTE this is not invoked yet, because we're still stuck with a bundle .exe -->
     <CustomAction Id="RemoveLegacyBundle"
               Directory="LocalAppDataFolder"
-              ExeCommand="[LocalAppDataFolder]\Package Cache\{a7cab659-0cbf-4b72-a658-b386d3d1d785}\Keybase_1.0.45-20180305114930+3a1c65a.386.exe /quiet /uninstall"
+              ExeCommand="[LocalAppDataFolder]\Package Cache\{896ea78e-a79b-4c72-890d-ad5e23ed036c}\Keybase_1.0.48-20180516084028+b56432f520.386.exe /quiet /uninstall"
```
Comment (jzila) [inline packaging/windows/WIXInstallers/KeybaseApps/KeybaseApps.wxs]: Did we not rename this to `2.0`?

File: shared/actions/gregor.js [hunk 2]
```diff
@@ -78,6 +78,7 @@ function checkReachabilityOnConnect() {
     RPCTypes.reachabilityStartReachabilityRpcPromise()
       .then(reachability => {
         dispatch(GregorGen.createUpdateReachability({reachability}))
+        logger.info('updating reachability')
```
Comment (jzila) [inline shared/actions/gregor.js]: Did you mean to leave this debugging in place?

**Comment by zanderz** [comment]
I've pulled the trigger on the server side, so this is ready, let's try it for a few builds before release

**Comment by zanderz** [comment]
This was pretty much cherry-picked from https://github.com/keybase/client/pull/10714, which was approved months ago, so I'm merging.

**Pull request merged**